internal/queue/loops/router/router.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright (c) 2025 Diagrid Inc.
+Licensed under the MIT License.
+*/
+
+package router
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"sync/atomic"
+
+	"github.com/dapr/kit/ptr"
+	"github.com/diagridio/go-etcd-cron/internal/api/queue"
+	"github.com/diagridio/go-etcd-cron/internal/queue/actioner"
+	"github.com/diagridio/go-etcd-cron/internal/queue/loops"
+	"github.com/diagridio/go-etcd-cron/internal/queue/loops/counters"
+	"github.com/go-logr/logr"
```

Comment:
nit on imports, particularly the `"github.com/go-logr/logr"` -> I think we want that in a new section above the `diagridio` one